<ui version="4.0" >
 <class>About</class>
 <widget class="QDialog" name="About" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>146</width>
       <height>168</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>170</width>
       <height>170</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>170</width>
       <height>170</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../resources/resources.qrc" >:/icon.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>157</width>
       <height>168</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="version_info" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Version %1
Released by %2 on %3</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QLabel" name="copyright_info" >
     <property name="text" >
      <string>This program uses Qt version 4.7.4 licensed under LGPL version 2.1.

Copyright %1. All rights reserved.

THE SOFTWARE IS PROVIDED "AS IS" WITH NO WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. IN NO EVENT SHALL COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, OR OTHER LIABILITY, WHETHER IN AN ACTION OF TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="close_bt" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
